geo/src/algorithm/affine_ops.rs

```diff
@@ -300,12 +316,37 @@ impl<T: CoordNum> AffineTransform<T> {
     /// The argument order matches that of the affine transform matrix:
     ///```ignore
     /// [[a, b, xoff],
-    /// [d, e, yoff],
-    /// [0, 0, 1]] <-- not part of the input arguments
+    ///  [d, e, yoff],
+    ///  [0, 0, 1]] <-- not part of the input arguments
     /// ```
     pub fn new(a: T, b: T, xoff: T, d: T, e: T, yoff: T) -> Self {
         Self([[a, b, xoff], [d, e, yoff], [T::zero(), T::zero(), T::one()]])
     }
+
+    /// Get the x-resolution value.
+    pub fn a(&self) -> T {
+        self.0[0][0]
+    }
+    /// Get the x-rotation value.
+    pub fn b(&self) -> T {
+        self.0[0][1]
+    }
+    /// Get the x-offset value.
+    pub fn xoff(&self) -> T {
+        self.0[0][2]
+    }
+    /// Get the y-rotation value.
+    pub fn d(&self) -> T {
+        self.0[1][0]
+    }
+    /// Get the y-resolution value.
+    pub fn e(&self) -> T {
+        self.0[1][1]
+    }
+    /// Get the y-offset value.
+    pub fn yoff(&self) -> T {
+        self.0[1][2]
+    }
```

Comment:
These doc comments – if they're needed at all – should not refer to the terminology ("resolution", "rotation" etc) used in Matt's blog post, as these methods aren't solely intended for vector <--> raster coordinate transformations.

Reply:
Yes, the methods here are only used to retrieve the element values, not for coordinate transformation. So what would be good descriptions for each of these getter methods? Do we simply don't assign any meaning to them, and assume that the user can figure out what the `a`, `b`, `xoff`, `c`, `d`, `yoff` values represent?

Reply:
That depends on whether it's reasonable to assume that someone who's trying to build a custom affine transform is familiar with its mechanics, and has (or is likely to) look at https://docs.rs/geo/latest/geo/algorithm/affine_ops/struct.AffineTransform.html, where the position and thus meaning of a, b, xoff etc are obvious.

Reply:
Ideally, there should be some description when a user hovers over the method in their IDE, so that they don't have to refer to the online documentation. Having a method with a single letter like `.a()` or `.b()` unfortunately isn't very descriptive...

Reply:
How about just a link back to the constructor docs, which has an ascii diagram of the matrix?

// See [new] for this value's role in the affine transformation.

Reply:
> How about just a link back to the constructor docs, which has an ascii diagram of the matrix?

Tried this locally, and clicking on the `AffineTransform::new` link in my editor still takes me to the online documentation page.

![Screenshot at 2024-04-26 16-42-12](https://github.com/georust/geo/assets/23487320/84a1a28a-f976-4a24-ae4f-c1ad04b87320)

But I'll take this as a good-enough compromise, unless anyone has any other suggestions?

Reply:
In the rendered docs online the link should point to the correct anchor.